web/gds-user-ui/src/components/TrixoQuestionnaireForm/index.tsx

```diff
@@ -99,30 +102,34 @@ const TrixoQuestionnaireForm: React.FC<TrixoFormProps> = ({
   }
 
   const handleNextStepClick = () => {
-    // const isValid = await isTrixoQuestionnaireValid(methods.getValues());
-    // console.log('[] isValid', isValid);
-    // if (!isValid) {
-    //   updateStepStatusToError(StepsIndexes.TRIXO_QUESTIONNAIRE);
-    // }
-    //
-    if (!isDirty) {
-      nextStep(data);
-    } else {
+    if (
+      isDirty ||
+      getMustComplyRegulationsFromData !== getMustComplyRegulations ||
+      getHasRequiredRegulatoryProgramFromData !== getHasRequiredRegulatoryProgram
+    ) {
       const payload = {
         step: StepEnum.TRIXO,
         form: {
           ...methods.getValues(),
           state: currentState()
         } as any
       };
+      console.log('[] isDirty  payload Trixo', payload);
```

Comment:
Should this be commented out? By the way, I think we make sure to comment out `console.log` statements used during debugging in a future sprint.